internal/pkg/service/buffer/config/conditions.go

```diff
@@ -1,14 +0,0 @@
-package config
-
-import (
-	"time"
-
-	"github.com/c2h5oh/datasize"
-)
-
-// Conditions struct configures slice upload and file import conditions.
-type Conditions struct {
```

Comment:
`config.Conditions` -> `model.Conditions`, aby som vyriesil zavislosti medzi pkgs.

Idealne by to malo byt v `condition.Conditions`, ale na to su potrebne dalsie zmeny v inych pkgs, ale casom sa to tam dostane.